packages/knip/src/index.ts

```diff
@@ -519,8 +519,10 @@ export const main = async (unresolvedConfiguration: CommandLineOptions) => {
                 line: exportedItem.line,
                 col: exportedItem.col,
               });
-              if (isType) fixer.addUnusedTypeNode(filePath, exportedItem.fixes);
-              else fixer.addUnusedExportNode(filePath, exportedItem.fixes);
+              if (issue) {
```

Comment:
Nitpick: since it's a boolean I'd like to name it `isAdded` or `isIssueAdded`

Reply:
Indeed! I fixed it to `isIssueAdded`
https://github.com/webpro-nl/knip/pull/654/commits/fffce7476772c824de42e04f7b99685db949e77d